submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/reactive/repositories/Repositories.kt

```diff
@@ -98,29 +97,31 @@ interface SubmissionRequestFilesRepository : CoroutineCrudRepository<DocSubmissi
         accNo: String,
         version: Int,
         index: Int,
-    ): Flux<DocSubmissionRequestFile>
+    ): Flow<DocSubmissionRequestFile>
 
     suspend fun getByPathAndAccNoAndVersion(path: String, accNo: String, version: Int): DocSubmissionRequestFile
 }
 
 interface FileListDocFileRepository : CoroutineCrudRepository<FileListDocFile, ObjectId> {
-    @Meta(cursorBatchSize = 100, flags = [org.springframework.data.mongodb.core.query.Meta.CursorOption.NO_TIMEOUT])
```

Comment:
I remember this was added in order to avoid the cursor timeout issue when processing large file lists. I don't think we should remove it unless it doesn't have any effect when using the reactive client

Reply:
we are not using long query now but paged query so this is not necesary.  That was the intention of this changes to remove long running cursor that may fail.